pkgs/development/mobile/ios-webkit-debug-proxy/default.nix

```diff
@@ -0,0 +1,58 @@
+{ stdenv
+, autoconf
+, automake
+, fetchFromGitHub
+, fetchpatch
+, lib
+, libimobiledevice
+, libusb1
+, libplist
+, libtool
+, openssl
+, pkg-config
+}:
+
+with lib;
```

Comment:
We want to get rid of this pattern, for context see: https://github.com/NixOS/nixpkgs/pull/209325

Reply:
woops sorry, I wasn't aware :grimacing: I moved the `with lib` to the `meta` section only, is that acceptable?

Reply:
Jup that's fine :D